### PR TITLE
Australian Cyber Security Packaging Changes

### PR DESCRIPTION
**Pull request by v-mchatla**
Australian Cyber Security Packaging Changes

   Required items, please complete
   
   Change(s):
   - See guidance below

   Reason for Change(s):
   - See guidance below

   Version Updated:
   - Required only for Detections/Analytic Rule templates
   - See guidance below

   Testing Completed:
   [Test Document](https://microsoftapc.sharepoint.com/:w:/r/teams/EcosystemCATTeam/_layouts/15/Doc.aspx?sourcedoc=%7BFEFD3A54-1B8A-440E-B366-96472DCFE668%7D&file=Sentinel%20Release%20Checklist%20-V2%20solutions-%20Australian%20Cyber%20Security%20Centre.docx&action=default&mobileredirect=true&share=IQFUOv3-ihsORLNmlkctz-ZoAdvJtNCi6N8rs1XXPK4DjJY)

-----------------------------------------------------------------------------------------------------------

### CODE REVIEW

**Comment by v-mchatla** [comment]
Packaging is completed, need to get it reviewed

**Comment by v-mchatla** [comment]
Hi @v-sabiraj,
Can you please approve and merge it if it is already reviewed.
Thanks

File: Solutions/Australian Cyber Security Centre/SolutionMetadata.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+	"publisherId": "azuresentinel",
+	"offerId": "azure-sentinel-solution-australiancybersecurity",
+	"firstPublishDate": "2022-11-23",
+	"providers": ["Australian Cyber Security Centre"],
+	"categories": {
+		"domains" : ["Security – Threat Intelligence"]
```
Comment (v-sabiraj) [inline Solutions/Australian Cyber Security Centre/SolutionMetadata.json:7]: Please check if we need to add "Security - Automation (SOAR)" as well, thanks.

Comment (v-mchatla) [reply Solutions/Australian Cyber Security Centre/SolutionMetadata.json:7]: Prateek only shared ["Security - Threat Intelligence"] domain category. Do you want me to check with Prateek once.
Thanks

**Review by v-sabiraj — APPROVED** [review]
looks good

**Pull request merged**